src/clr-angular/utils/conditional/if-open.service.ts

```diff
@@ -83,4 +83,18 @@ export class IfOpenService {
         this.open = !this.open;
         delete this.originalEvent;
     }
+
+    /**
+     *  Popovers might need to ignore click events on an element
+     *  (eg: popover opens on focus on an input field. Clicks should be ignored in this case)
+     */
+    private _ignoredElementChange: Subject<ElementRef> = new Subject<ElementRef>();
+
+    get ignoredElementChange(): Observable<ElementRef> {
```

Comment:
I'm guessing you need an Observable because this is set too late in the lifecycle of the select menu?

Reply:
yes